### ISSUE TITLE
Support Bash on Windows

### ISSUE THREAD

**Issue by metcalfc** [ahmetb/kubectx#97]
I'm using a bash on Windows. `kubectl` is on the path but as `kubectl.exe`. I've tried a number of things on my side to get around it (alias, shell hackery, etc) to no avail. 

I've submitted a potential fix in https://github.com/ahmetb/kubectx/pull/96. Thank you.

**Comment by metcalfc** [comment]
I did think of a mildly evil alternative. I didn't want to have to go through the Windows path and link all the exe binaries by hand. It would be error-prone and tedious. However, if it was automated maybe it wouldn't be as bad. So I hijacked bash's `command_not_found_handle` to find the exe binaries and link them into a `~/bin` directory. I verified this does work as well.

```eval "original_$(declare -f command_not_found_handle)"
command_not_found_handle () {
    if command -v "$1".exe > /dev/null 2>&1; then
        ln -s "$(which ${1}.exe)" ~/bin/${1}
        "$1".exe "${@:2}";
        return $?;
    else
        original_command_not_found_handle "$@"
    fi
}
```

It would be nice if you didn't have to result to hijacking and binary linking. But if the patch isn't something you'd want to take then folks can find a work around here.
